packages/core/.eslintrc.cjs

```diff
@@ -11,4 +11,7 @@ module.exports = {
     browser: true,
     node: true,
   },
+  rules: {
+    'sonarjs/no-duplicate-string': 'off',
```

Comment:
Hopefully this doesn't start The Great String Constant Debate, but I generally feel like module-level string constants are overkill. 🤷🏼